tests/func/test_remote.py

```diff
@@ -169,34 +170,11 @@ def test(self):
         self.assertEqual(ret, 0)
 
 
-def test_large_dir_progress(repo_dir, dvc_repo):
```

Comment:
I believe this test is unnecessary. When it was created, progress was triggered only for large dir. Now it is triggered always. If it is to stay, there is no need for 100 files in dir, 1 will be enough.